### DISCUSSION TITLE
abp extra property dictionary is set to empty when a user update there  password

### DISCUSSION THREAD

**Discussion by mabdalla123** [abpframework/abp#21130] [Q&A]
i have noticed on my application when a user update his password the ExtraProps are set to {}
```
 private static void ConfigureExtraProperties()
 {
     /* You can configure extra properties for the
      * entities defined in the modules used by your application.
      *
      * This class can be used to define these extra properties
      * with a high level, easy to use API.
      *
      * Example: Add a new property to the user entity of the identity module
     */
     ObjectExtensionManager.Instance.Modules()
        .ConfigureIdentity(identity =>
        {
            identity.ConfigureUser(user =>
            {
                user.AddOrUpdateProperty<Guid>( //property type: string
                    "BranchId", //property name
                    property =>
                    {
                        //validation rules
                        property.UI.Lookup.Url = "/api/app/branch";
                        property.UI.Lookup.DisplayPropertyName = "nameAR";


                        //...other configurations for this property
                    }
                );
            });
        });
     /*

      * See the documentation for more:
      * https://docs.abp.io/en/abp/latest/Module-Entity-Extensions
      */
 }
```

i was following the docs and this is how i added the extra prop

please not that the user is not allowed to update this value only the admin

**Comment by maliming** [comment]
hi

- Your ABP Framework version.
- Your User Interface type (Angular/MVC/React... etc.) if the issue is related to a specific UI
- Your database provider(EF Core/MongoDB)
- Please share the steps and code to reproduce the problem.

**Reply by mabdalla123** [reply]
the project currently uses 8.2 version but the was present in 7.4 
User interface is MVC
Database is Entity Framework Core -> Microsoft sql server 
i have added the extra prop using the prevoiuse code , when a user change there password  on the change password screen  , this prop is set to "{}"

**Reply by maliming** [reply]
hi

I can't reproduce this in a new template project.
